media-api/app/controllers/MediaApi.scala

```diff
@@ -306,12 +307,29 @@ object MediaApi extends Controller with ArgoHelpers {
     Link("related-labels", uri)
   }
 
+  private val getSuggestedLabelsLink =
+    Link("suggested-labels", s"$rootUri/suggest/edits/labels")
+
   def suggestMetadataCredit(q: Option[String], size: Option[Int]) = Authenticated.async { request =>
     ElasticSearch
       .completionSuggestion("suggestMetadataCredit", q.getOrElse(""), size.getOrElse(10))
       .map(c => respondCollection(c.results))
   }
 
+  def suggestLabels() = Authenticated {
+    val pseudoFamousLabels = List(
```

Comment:
Yea, I think the point would be to have "leading" which are guardian specific, then, as they become the most popular labels, use the standard labels suggester.